src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -0,0 +1,32 @@
+<%@ tag description="instructorFeedbacks and instructorHome - Feedback Session publish/unpublish button" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
+<%@ attribute name="buttonType" required="true" %>
+<c:choose>
+    <c:when test="${publishButton.hasUnpublish}">
+        <a class="btn ${buttonType} btn-tm-actions session-unpublish-for-test"
+           href="${publishButton.unpublishLink}"
+           title="<%= Const.Tooltips.FEEDBACK_SESSION_UNPUBLISH %>"
+           data-toggle="tooltip"
+           data-placement="top"
+           onclick="return toggleUnpublishEvaluation(${publishButton.toggleUnpublishSessionParams});"
+           <c:if test="${not publishButton.allowedToUnpublish}">disabled="disabled"</c:if>>
+            Unpublish Results
+        </a>
+    </c:when>
+    <c:otherwise>
+        <div title="${publishButton.tooltipText}"
+             data-toggle="tooltip"
+             data-placement="top"
+             style="display: inline-block; padding-right: 5px;">
+            <a class="btn ${buttonType} btn-tm-actions session-publish-for-test<c:if test="${not publishButton.hasPublish}"> disabled</c:if>"
+               href="${publishButton.publishLink}"
+               <c:if test="${publishButton.hasPublish}">onclick="return togglePublishEvaluation(${publishButton.togglePublishSessionParams});"</c:if>
+               <c:if test="${not publishButton.hasPublish}">onclick="return false"</c:if>
+               <c:if test="${not publishButton.allowedToPublish}">disabled="disabled"</c:if>>
```

Comment:
I'm not sure whether this should be done in this PR or another issue should be open to treat this :\

The class "disabled" does exactly the same thing as disabled="disabled".

disabled="disabled" is valid XHTML but disabled is enough in HTML.

Is there a better way to check for both booleans and see if it should be disabled and disable it just once and in a consistent manner?